src/components/WorkerPoolQueueUpcomingFlowRunsList.vue

```diff
@@ -0,0 +1,45 @@
+<template>
+  <div class="worker-pool-queue-upcoming-flow-runs-list">
+    <FlowRunList :selected="[]" :flow-runs="filteredFlowRuns" disabled />
+
+    <p-empty-results v-if="empty">
+      <template v-if="isPaused" #message>
+        <p-icon class="work-queue-flow-runs-list__icon" icon="PauseIcon" />
+        This worker pool queue is paused and will not submit runs
+      </template>
+      <template v-else #message>
+        No upcoming runs
+      </template>
+    </p-empty-results>
+  </div>
+</template>
+
+<script lang="ts" setup>
+  import { useSubscription } from '@prefecthq/vue-compositions'
+  import { computed, toRefs, watch } from 'vue'
+  import { FlowRunList } from '@/components'
+  import { useWorkspaceApi } from '@/compositions'
+  import { WorkerPoolQueue } from '@/models'
+
+  const props = defineProps<{
+    workerPoolName: string,
+    workerPoolQueue: WorkerPoolQueue,
+  }>()
+
+  const api = useWorkspaceApi()
+
+  const { workerPoolName } = toRefs(props)
+  const { workerPoolQueue } = toRefs(props)
+
+  const workerPoolScheduledRunsSubscription = useSubscription(api.workerPools.getWorkerPoolScheduledRuns, [workerPoolName.value, { workerPoolQueueNames: [workerPoolQueue.value.name] }], { interval: 30000 })
```

Comment:
This interval might be a little long for this - it'd be good to at least match the polling interval, which is 10 seconds

Reply:
Feel free to disregard ^ if that doesn't make sense for worker pools though

Reply:
Is polling interval always 10 seconds?